lwt/jbuild

```diff
@@ -0,0 +1,8 @@
+(library
+ ((name        conduit_lwt)
+  (public_name conduit-lwt)
+  (preprocess (pps (ppx_sexp_conv)))
+  (wrapped false)
+  (modules (resolver_lwt))
+  (libraries (conduit lwt))))
+(jbuild_version 1)
```

Comment:
can you put the `(jbuild_version 1)` stanza at the beginning of the `jbuild` file? It applies to all `jbuild` files :-)